Packs/SecurityTrails/Integrations/SecurityTrails/SecurityTrails.py

```diff
@@ -0,0 +1,411 @@
+        outputs={"domain_dns_history": records},
+        readable_output=tableToMarkdown(f"Domain DNS History for {domain}", records),
+    )
+
+    return results
+
+
+def get_domain_whois_history(client: Client, **args) -> CommandResults:
+    domain = args.get("domain")
+
+    params = {"page": int(args.get("page", 0))}
+
+    uri = f"/history/{domain}/whois"
+    response = client._http_request("GET", uri, params=params)
+    records = response["result"].get("items")
+
+    results = CommandResults(
+        outputs_prefix="SecurityTrails",
+        outputs_key_field=f"SecurityTrails.domain_whois_history",
+        outputs={"domain_whois_history": records},
+        readable_output=tableToMarkdown(f"Domain Whois History for {domain}", records),
+    )
+
+    return results
+
+
+def get_ip_neighbors(client: Client, **args) -> CommandResults:
+    ip = args.get("ip")
+
+    uri = f"/ips/nearby/{ip}"
+    response = client._http_request("GET", uri)
+    records = response.get("blocks")
+
+    results = CommandResults(
+        outputs_prefix="SecurityTrails",
+        outputs_key_field=f"SecurityTrails.ip_neighbors",
+        outputs={"ip_neighbors": records},
+        readable_output=tableToMarkdown(f"IP Neighbors for {ip}", records),
+    )
+
+    return results
+
+
+def search_domain_with_dsl(client: Client, **args) -> CommandResults:
+    params = {
+        "include_ips": args.get("include_ips", "false"),
+        "page": int(args.get("page", 0)),
+        "scroll": str(args.get("scroll", "false")),
+    }
+
+    body = {"query": str(args.get("query"))}
+
+    uri = f"/domains/list"
+    response = client._http_request("POST", uri, params=params, json_data=body)
+    records = response.get("records")
+
+    results = CommandResults(
+        outputs_prefix="SecurityTrails",
+        outputs_key_field=f"SecurityTrails.domain_dsl_search",
+        outputs={"dsl_search_domain": records},
+        readable_output=tableToMarkdown(
+            f"DSL Search: {str(args.get('query'))}", records
+        ),
+    )
+
+    return results
+
+
+def search_domain_with_dsl_statistics(client: Client, **args) -> CommandResults:
+    params = {
+        "include_ips": args.get("include_ips", "false"),
+        "page": int(args.get("page", 0)),
+        "scroll": str(args.get("scroll", "false")),
+    }
+
+    body = {"query": str(args.get("query"))}
+
+    uri = f"/domains/stats"
+    response = client._http_request("POST", uri, params=params, json_data=body)
+
+    results = CommandResults(
+        outputs_prefix="SecurityTrails",
+        outputs_key_field=f"SecurityTrails.domain_dsl_search",
+        outputs={"domain_statistics": response},
+        readable_output=tableToMarkdown(
+            f"Statistics for Query: {str(args.get('query'))}", response
+        ),
+    )
+
+    return results
+
+
+def search_ip_with_dsl(client: Client, **args) -> CommandResults:
+    params = {"page": int(args.get("page"))}
```

Comment:
mypy is failing this line, Please add a default value like the rest of the commands

Reply:
added default value of 1 for page